src/pages/Admin/Charity/EditProfile/Form.tsx

```diff
@@ -121,6 +122,8 @@ export default function Form() {
           label="Website of your organization"
           placeholder="https://website.org"
         />
+
+        <HideBGTipCheckbox />
```

Comment:
It can go here for now, as we don't as many specific design reqs for this feature. Just need to get it live for now and we can always shuttle it elsewhere based on operations/design/users feedback dictates.